wdl/downsampling_pipeline_part1_dynamic.wdl

```diff
@@ -0,0 +1,154 @@
+version 1.0
+
+workflow downSampling_01 {
+
+    #################################################################################
+    ####        Required basic arguments for downsampling part 1                    #
+    #################################################################################
+    
+    input {
+        File bam_or_cram_file
+        File reference_fasta
+        Int? cram_to_bam_disk_size
+        String? cram_to_bam_mem_size
+        String downsample_docker
+
+        Boolean is_bam_ = basename(bam_or_cram_file, ".bam") + ".bam" == basename(bam_or_cram_file)
```

Comment:
Why is is_bam an input? It seems like it should just be a variable that's determined within the WDL using the logic you've already written